ccc/ccc.go

```diff
@@ -47,11 +48,11 @@ var sortingRegexp = regexp.MustCompile("_(.+)")
 
 // Simple handler to handle fatal errors
 func fatal(err error) {
-	fmt.Fprintf(os.Stderr, "%s: %s\n", os.Args[0], err)
+	_, _ = fmt.Fprintf(os.Stderr, "%s: %s\n", os.Args[0], err)
 	os.Exit(1)
 }
 
-// Console writer to capture console output
+// ConsoleWriter Console writer to capture console output
```

Comment:
You should decide which one you want, with or without blank

Reply:
I think, ConsoleWriter is the data type there. IntelliJ suggested to write it like this. I could reformat it "ConsoleWriter object to capture console output". Deal?

Reply:
It's redundant information since eg. an object called 'ConsoleWriter' should not read smth from a file.'
And, to be precise, `type struct` is the go variant of a class, so I'd omit the word object.
Proceed as you like.

Reply:
Sorry. I don't get what you  wrote there. 😆 

So the ConsoleWriter type there extends the `type struct` type (a class as you already said) and in the next line it adds a Write method, which captures the stdout and stderr (where it is used). This is what I figured out how  "memory streams" work in Golang.

The doc comment there should be prefixed by the type that it describes (according to IntelliJ), so the prefix is required.

What did you mean with "should not read smth from a file" and "word object"?

Reply:
1. 'should not read from a file' was meant as a negative example. It would be at least a code smell if the ConsoleWriter would read from a file instead of writing to the console. It's one of the basic recommendations to use self explanatory names for classes, objects, methods and so on. So if the ConsoleWriter does not do some additional fancy stuff besides writing to the console no further documentation is necessary.
2. You uses the word object in your suggestion _"ConsoleWriter object to capture console output"_. ConsoleWiter is NOT an object, it's a class construct. The object is created in line 80...

Reply:
Ahhh okay. So maybe just: "ConsoleWrite used to capture console output"?